backend/src/Infrastructure/Dependencies.cs

```diff
@@ -20,10 +22,22 @@ public static void ConfigureServices(IConfiguration configuration, IServiceColle
                 );
 
             // Add Azure AD B2C authentication to the ASP.NET Core pipeline
-            services
-                .AddMicrosoftIdentityWebApiAuthentication(configuration, "AzureAdB2C");
+            services.AddSingleton<IAuthenticationProvider>(o =>
+            {
+                var confidentialClientApplication = ConfidentialClientApplicationBuilder
+                    .Create(configuration.GetSection("AzureAdB2C")["ClientId"])
+                    .WithTenantId(configuration.GetSection("AzureAdB2C")["TenantId"])
+                    .WithClientSecret(configuration.GetSection("AzureAdB2C")["ClientSecret"])
+                    .Build();
+
+                return new Microsoft.Graph.Auth.ClientCredentialProvider(confidentialClientApplication);
+            });
+
+            services.AddSingleton(conf => new GraphServiceClient(conf.GetService<IAuthenticationProvider>()));
+
+            services.AddSingleton(o => new ExtensionPropertyNameBuilder(configuration.GetSection("AzureAdB2C")["ExtensionAppId"]));
 
-            JwtSecurityTokenHandler.DefaultMapInboundClaims = false;
```

Comment:
Bez tego nie działało mapowanie tokenów. Działało lokalnie u Ciebie bez tego, czy te zmiany wyżej zmieniają to?

Reply:
działało bez tego z nową konfiguracją do Grapha